### ISSUE TITLE
Profiler is not working

### ISSUE THREAD

**Issue by tcoenraad** [danhper/fundle#9]
When I use `fundle init --profile` in my `config.fish` to profile fundle gives me lot of warnings like this:

``` fish
(standard_in) 1: illegal character: N
(standard_in) 1: illegal character: N
__fundle_load_plugin tuvistavie/oh-my-fish-core /Users/twancoenraad/.config/fish/fundle 1:
```

Can you look into this?

**Comment by danhper** [comment]
@tcoenraad Thanks for the report. I did not know that GNU date and BSD date were so different.
This should be fixed on 0.3.2, but you need to have `gdate` installed. `brew install coreutils` should install it, as I wrote it in the readme.
I took the time to add OSX build to travis, so I should be able to avoid those compatibility issues now.

**Comment by tcoenraad** [comment]
It is fixed, thank you :+1:
